tool/src/index.js

```diff
@@ -0,0 +1,42 @@
+var xlsxtojson = require("xlsx-to-json");
```

Comment:
This file is not es6 and does not follow the standard-js code style.

I'm okay with leaving this as-is, since it's a dev tool only, just wanted to mention it.

Reply:
thanks